src/isar/models/communication/queues/queues.py

```diff
@@ -7,11 +7,26 @@
 
 class Queues:
     def __init__(self) -> None:
-        self.start_mission: QueueIO = QueueIO(input_size=1, output_size=1)
-        self.stop_mission: QueueIO = QueueIO(input_size=1, output_size=1)
-        self.pause_mission: QueueIO = QueueIO(input_size=1, output_size=1)
-        self.resume_mission: QueueIO = QueueIO(input_size=1, output_size=1)
-        self.single_action: QueueIO = QueueIO(input_size=1, output_size=1)
+        self.api_start_mission: QueueIO = QueueIO(input_size=1, output_size=1)
+        self.api_stop_mission: QueueIO = QueueIO(input_size=1, output_size=1)
```

Comment:
Is there any way for a user of ISAR that have queued a mission, to see that the mission is in the queue? (nothing to do with this line of code, just put the question here)

Reply:
No, once the state machine consumes the event the API thread can no longer track it, but that is not some behaviour which has changed in the PR.

Reply:
I agree, more of a curiosity from my side.